api/src/main/java/javax/enterprise/context/spi/Context.java

```diff
@@ -58,7 +58,7 @@
      * 
      * @return the scope
      */
-    public Class<? extends Annotation> getScope();
```

Comment:
I believe this is just a clean up. As you said they are public "by default" so there is no need to explicitly state it.
My guess is that John's IDE was going nuts seeing all those modifiers :)